dispatch/dispatch.go

```diff
@@ -81,6 +81,7 @@ type AlertBlock struct {
 type APIAlert struct {
 	*model.Alert
 	Status types.AlertStatus `json:"status"`
+	ID     string            `json:"id"`
```

Comment:
As the fingerprint is not really an id, I would prefer renaming this to `HASH`. What do you think?